package.json

```diff
@@ -1687,10 +1687,11 @@
     "cssnano": "^5.1.12",
     "cssnano-preset-default": "^5.2.12",
     "csstype": "^3.0.2",
-    "cypress": "13.6.3",
+    "cypress": "13.15.0",
     "cypress-axe": "^1.5.0",
     "cypress-file-upload": "^5.0.8",
     "cypress-multi-reporters": "^1.6.4",
+    "cypress-network-idle": "^1.14.2",
```

Comment:
Instead of relying on the global loading indicator being present for the alerts based tests, we'll instead wait for network requests to complete on the page (this helps avoid flakiness due to any loading indicator flicker)